pkg/util/dirs.go

```diff
@@ -125,17 +125,29 @@ func JXBinLocation() (string, error) {
 
 // JXBinaryLocation Returns the path to the currently installed JX binary.
 func JXBinaryLocation(commandInterface Commander) (string, error) {
```

Comment:
so this no longer uses the Commander, and the tests seem to serve no purpose so can be simplified and removed.
that does leave me wondering how best to test this however.  @i0n do you have any ideas?